### PR TITLE
create text for home page

### PR DESCRIPTION
**Pull request by drouba**

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -29,12 +29,230 @@
...
+      <h4>Students</h4>
+      Upload student lists to specific sections through the course lecture pages of the lectures you are assigned to
+    </li><br>
+    <li>
+      <h4>Marked Entities</h4>
+      View all marked entities for your lectures
+      <br>Create marked entities for your lectures
+      <br>Update marked entities for your lectures
+      <br>Comment on students uploaded files for your lectures' marked entities
+      <br>Create Discussions related to marked entities visible only to TAs and instructors
```
Comment (jamestjw) [inline public/index.php:138]: TAs instructors **and students**

File: public/index.php [hunk 2]
```diff
@@ -29,12 +29,230 @@
...
+  </li><br>
+  <li>
+    <h4>Students</h4>
+    View all students registered in lectures associated to your course sections
+  </li><br>
+  <li>
+    <h4>Marked Entities</h4>
+    View all marked entities for your lectures
+    <br>Comment on students uploaded files for your lectures' marked entities
+    <br>Create Discussions related to marked entities visible only to TAs and instructors
```
Comment (jamestjw) [inline public/index.php:193]: here too

File: public/index.php [hunk 3]
```diff
@@ -29,12 +29,230 @@
...
+    <h4>Students</h4>
+    View all students registered in lectures associated to your course sections
+  </li><br>
+  <li>
+    <h4>Marked Entities</h4>
+    View all marked entities for your lectures
+    <br>Comment on students uploaded files for your lectures' marked entities
+    <br>Create Discussions related to marked entities visible only to TAs and instructors  
+    <br>Reply to messages on marked entity Discussions
+    <br>Create polls in marked entiy discussion messages
```
Comment (jamestjw) [inline public/index.php:195]: typo here

File: public/index.php [hunk 4]
```diff
@@ -29,12 +29,230 @@
...
+    View your internal emails 
+    <br>Send emails internally to other users
+  </li><br>
+  <li>
+    <h4>Meetings</h4>
+    View all meetings created by students in your teams
+  </li><br>
+  <li>
+    <h4> Courses, Lectures and Sections</h4>
+    View the lecture pages for lectures who's section you are enrolled in
```
Comment (jamestjw) [inline public/index.php:233]: *whose section I think?

File: public/index.php [hunk 5]
```diff
@@ -29,12 +29,230 @@
...
+    <h4>Students</h4>
+    View all students registered in lectures associated to course sections you are enrolled in
+  </li><br>
+  <li>
+    <h4>Marked Entities</h4>
+    View all marked entities for your lectures associated to course sections you are enrolled in
+    <br>Upload files to marked entities
+    <br>Create Discussions related to marked entities 
+    <br>Reply to messages on marked entity Discussions
+    <br>Create polls in marked entiy discussion messages
```
Comment (jamestjw) [inline public/index.php:245]: same typo

File: public/index.php [hunk 6]
```diff
@@ -29,12 +29,230 @@
...
+  </li><br>
+  <li>
+    <h4>emails</h4>
+    View your internal emails 
+    <br>Send emails internally to other users
+  </li><br>
+  <li>
+    <h4>Meetings</h4>
+    View all meetings created by students in your teams
+  </li><br>
```
Comment (jamestjw) [inline public/index.php:230]: need to mention starting meetings etc ?

File: public/index.php [hunk 7]
```diff
@@ -29,12 +29,230 @@
...
+    <li>
+      <h4>Teams</h4>
+      View teams for all lectures 
+    </li><br>
+    <li>
+      <h4>Account settings</h4>
+      Change the Admin account settings
+    </li><br>
+    <li>
+      <h4>emails</h4>
```
Comment (jamestjw) [inline public/index.php:54]: Capital e probably?

File: public/index.php [hunk 8]
```diff
@@ -29,12 +29,230 @@
...
+  <li>
+    <h4>Teams</h4>
+    View teams for your lectures associated to your assigned sections
+  </li><br>
+  <li>
+    <h4>Account settings</h4>
+    Change your account settings
+  </li><br>
+  <li>
+    <h4>emails</h4>
```
Comment (chris7neves) [inline public/index.php:173]: e here as well

File: public/index.php (hunk 2 above)
Comment (drouba) [reply public/index.php:193]: I checked though and on marked entities when I create a discusssion as a TA that discussion isnot visible to students

File: public/index.php (hunk 3 above)
Comment (drouba) [reply public/index.php:195]: good catch

File: public/index.php (hunk 4 above)
Comment (drouba) [reply public/index.php:233]: ah yes sorry

File: public/index.php (hunk 6 above)
Comment (drouba) [reply public/index.php:230]: oh yeah good point

File: public/index.php (hunk 7 above)
Comment (drouba) [reply public/index.php:54]: yup

File: public/index.php (hunk 8 above)
Comment (drouba) [reply public/index.php:173]: yup

**Review by jamestjw — APPROVED** [review]

**Pull request merged**